sentinel.py

```diff
@@ -4,31 +4,50 @@
 """
 Create sentinel and singleton objects.
 
-Copyright 2014 © Eddie Antonio Santos. MIT licensed.
+ Copyright 2014 © Eddie Antonio Santos. MIT licensed.
+
+ This software is released under the MIT License.
+ https://opensource.org/licenses/MIT
 """
 
 import inspect
+from typing import Dict, Tuple, Optional, Any
 
 __all__ = ['create']
-__version__ = '0.1.2'
+__version__ = '0.1.3'
 
+class _SentinelMeta(type):
+    def __call__(cls, *args:Tuple[Any], **kwargs:Dict[str, Any]) -> object:
+        inst = _sinstances.get(cls)
+        if inst is None:
+            inst = type.__call__(cls, *args, **kwargs)
+            _sinstances[cls] = inst
 
-def get_caller_module():
-    """
-    Returns the name of the caller's module as a string.
+        return inst
```

Comment:
Is it not possible to define the `property` here?

```suggestion
        return inst
        
    @property
    def instance(self):
        "Gets the instance of this sentinel type."
        return _sinstances.get(self)
```